### PR TITLE
[ASP.NET Core] minor format fix, better null check

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

- minor format fix
- better null check (ref: https://github.com/OpenAPITools/openapi-generator/pull/2768)

cc @mandrean (2017/08), @jimschubert (2017/09)

### CODE REVIEW

**Comment by wing328** [comment]
Also cc @A-Joshi who's been making enhancements to the aspnetcore generator.

File: modules/openapi-generator/src/main/resources/aspnetcore/2.0/model.mustache [hunk 1]
```diff
@@ -92,6 +92,7 @@ namespace {{packageName}}.Models
                 (
                     {{name}} == other.{{name}} ||
                     {{name}} != null &&
+                    other.{{name}} != null &&
```
Comment (jimschubert) [inline modules/openapi-generator/src/main/resources/aspnetcore/2.0/model.mustache]: Good catch. `SequenceEqual` method delegates to the static [Enumerable.SequenceEqual](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=netframework-4.8#System_Linq_Enumerable_SequenceEqual__1_System_Collections_Generic_IEnumerable___0__System_Collections_Generic_IEnumerable___0__System_Collections_Generic_IEqualityComparer___0__) which will throw an argument null exception when this input is null.

File: modules/openapi-generator/src/main/resources/aspnetcore/2.0/controller.mustache [hunk 2]
```diff
@@ -1,8 +1,10 @@
 {{>partial_header}}
 using System;
 using System.Collections.Generic;
-using Microsoft.AspNetCore.Mvc;{{#useSwashbuckle}}
-using Swashbuckle.AspNetCore.SwaggerGen;{{/useSwashbuckle}}
+using Microsoft.AspNetCore.Mvc;
+{{#useSwashbuckle}}
```
Comment (jimschubert) [inline modules/openapi-generator/src/main/resources/aspnetcore/2.0/controller.mustache]: This adds a newline, which I don't think is intentional.

Comment (wing328) [reply modules/openapi-generator/src/main/resources/aspnetcore/2.0/controller.mustache]: I think the version of jmustache has better handling of newline: https://github.com/samskivert/jmustache#newline-trimming

**Pull request merged**